README.md

```diff
@@ -99,7 +100,7 @@ Imagine that you take the code from this question, save it to a file called `sus
 
 What would be the output? Explain your answer.
 
-<!-- Replace this comment with your answer -->
+"sushi is tasty" the var is higher than 0, so it would be complete and wouldn't continue to the else if, and the I don't like sushi would appear if the var is assigned something that isn't >0 or >= 3.<!-- Replace this comment with your answer -->
```

Comment:
Again, you've given the right answer - that "sushi is tasy" will be what's printed - but your explanation isn't correct. 

If this program reached line 92 (`} else if (x >= 3) {`), it would error out because `x` isn't defined anywhere. But because it's part of an `if...else`, and the first condition is true, _we will never check the second condition_. As a result, even though line 92 would break our code, it never gets executed, so we don't hit an error.